test-infra/prow/config.yaml

```diff
@@ -137,24 +137,42 @@ periodics:
    agent: kubernetes
    spec:
     containers:
-    - image: gcr.io/compute-image-tools-test/test-runner:latest
-      args:
-      - "-out_path=/artifacts/junit.xml"
-      # One project is enough for daisy tests, no test requires a write lock.
-      - "-projects=compute-image-test-pool-001"
-      - "-zone=us-central1-c"
-      - "daisy_integration_tests/daisy_e2e.test.gotmpl"
-      env:
-      - name: REPO_OWNER
-        value: GoogleCloudPlatform
-      - name: REPO_NAME
-        value: compute-image-tools
-      - name: ARTIFACTS
-        value: /artifacts
-      volumeMounts:
-      - name: compute-image-tools-test-service-account
-        mountPath: /etc/compute-image-tools-test-service-account
-        readOnly: true
+      - image: gcr.io/compute-image-tools-test/test-runner:latest
+        args:
+          - "-out_path=/artifacts/junit.xml"
+          # One project is enough for daisy tests, no test requires a write lock.
+          - "-projects=compute-image-test-pool-001"
+          - "-zone=us-central1-c"
+          - "daisy_integration_tests/daisy_e2e.test.gotmpl"
+        env:
+          - name: REPO_OWNER
+            value: GoogleCloudPlatform
+          - name: REPO_NAME
+            value: compute-image-tools
+          - name: ARTIFACTS
+            value: /artifacts
+        volumeMounts:
+          - name: compute-image-tools-test-service-account
+            mountPath: /etc/compute-image-tools-test-service-account
+            readOnly: true
+      - image: gcr.io/compute-image-tools-test/test-runner:latest
+        args:
+          - "-out_path=/artifacts/junit.xml"
+          # One project is enough for daisy tests, no test requires a write lock.
+          - "-projects=compute-image-test-pool-006"
+          - "-zone=us-central1-c"
+          - "daisy_integration_tests/daisy_e2e_quota_exceeded.test.gotmpl"
```

Comment:
Does this imply that you've edited 006 to have a limited amount of quota?

Reply:
No, I filed ticket to reduce quota but I was told quota can't be decreased. So I have to set a big number that is greater than the default quota.